File: crates/deslop/tests/fixtures/csharp-type3/Epsilon.cs
namespace Epsilon
{
    public class Worker
    {
        public int Process(int limit)
        {
            if (limit < 0)
            {
                return 0;
            }
            int accumulator = 0;
            for (int cursor = 0; cursor < limit; cursor = cursor + 1)
            {
                accumulator = accumulator + cursor;
            }
            return accumulator;
        }
    }
}

File: crates/deslop/tests/fixtures/dart-type3/epsilon.dart
int aggregate(int limit) {
  if (limit < 0) {
    return 0;
  }
  var accumulator = 0;
  for (var cursor = 0; cursor < limit; cursor = cursor + 1) {
    accumulator = accumulator + cursor;
  }
  return accumulator;
}
